modules/picard/createsequencedictionary/main.nf

```diff
@@ -31,8 +31,8 @@ process PICARD_CREATESEQUENCEDICTIONARY {
         -Xmx${avail_mem}g \\
         CreateSequenceDictionary  \\
         $args \\
-        R=$fasta \\
-        O=${prefix}.dict
+        -R $fasta \\
+        -O ${prefix}.dict
```

Comment:
Can we use long name parameters everywhere?

Reply:
Fine by me! I'll convert them all

Reply:
I did the same for GATK not so long ago.
Thank you for doing that here

Reply:
Perhaps it's something that can be added to the guidelines?  Mandatory long params for all tools (where applicable).

Reply:
Good idea